src/main.rs

```diff
@@ -23,6 +23,7 @@ fn main() -> color_eyre::Result<()> {
     run_app(&mut terminal, &mut app)?;
     utils::restore()?;
 
+    Logger::info("patch-hub finished");
```

Comment:
Nitpick: this shouldn't pertain to this commit. However, I don't mind merging it this way :grimacing: